### PR TITLE
Use translation-google module instead of Google API

### PR DESCRIPTION
**Pull request by diogoscf**
Allows users to use this bot without paying (smh)

https://github.com/wilsonwu/translation-google

### CODE REVIEW

**Comment by scoopgracie** [comment]
Or use Apertium

@diogoscf: 😀

File: src/bot_telegram.ts [hunk 1]
```diff
@@ -37,9 +37,7 @@ bot.onText(/\/except (.+)/, async (msg, match) => {
 
   const chatMember = await bot.getChatMember(chatId, userId)
 
-  if (isAdminUser(chatMember)) {
-    // okay
-  } else {
+  if (!isAdminUser(chatMember)) {
```
Comment (diogoscf) [inline src/bot_telegram.ts:40]: Also fixed this, because wtf @bartekpacia

Comment (bartekpacia) [reply src/bot_telegram.ts:40]: 🤔
"that's in case I wanted to do something when the user is admin (in the future)"

File: .env.sample [hunk 2]
```diff
@@ -1,9 +1,8 @@
 TOKEN=
-GCP_API_KEY=
```
Comment (bartekpacia) [inline .env.sample:2]: Please bring it back

Comment (diogoscf) [reply .env.sample:2]: It wouldn't be necessary now

File: src/bot_telegram.ts (hunk 1 above)
Comment (diogoscf) [reply src/bot_telegram.ts:40]: Yeah right

Comment (diogoscf) [reply src/bot_telegram.ts:40]: You can always edit back if you need to

File: .env.sample (hunk 2 above)
Comment (bartekpacia) [reply .env.sample:2]: just add it, it serves as a template

File: src/bot_telegram.ts (hunk 1 above)
Comment (bartekpacia) [reply src/bot_telegram.ts:40]: sure master

File: .env.sample (hunk 2 above)
Comment (diogoscf) [reply .env.sample]: Alright it's back

**Pull request merged**